docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,490 @@
+`FlightDelay`. Each document in the dataset contains details for a single 
+flight, so this data is ready for analysis as it is already in a two-dimensional 
+entity-based data structure (_{dataframe}_). In general, you often need to 
+{ref}/transforms.html[transform] the data into an entity-centric index before 
+you analyze the data.
+
+This is an example source document from the dataset:
+
+```
+{
+  "_index": "kibana_sample_data_flights",
+  "_type": "_doc",
+  "_id": "S-JS1W0BJ7wufFIaPAHe",
+  "_version": 1,
+  "_seq_no": 3356,
+  "_primary_term": 1,
+  "found": true,
+  "_source": {
+    "FlightNum": "N32FE9T",
+    "DestCountry": "JP",
+    "OriginWeather": "Thunder & Lightning",
+    "OriginCityName": "Adelaide",
+    "AvgTicketPrice": 499.08518599798685,
+    "DistanceMiles": 4802.864932998549,
+    "FlightDelay": false,
+    "DestWeather": "Sunny",
+    "Dest": "Chubu Centrair International Airport",
+    "FlightDelayType": "No Delay",
+    "OriginCountry": "AU",
+    "dayOfWeek": 3,
+    "DistanceKilometers": 7729.461862731618,
+    "timestamp": "2019-10-17T11:12:29",
+    "DestLocation": {
+      "lat": "34.85839844",
+      "lon": "136.8049927"
+    },
+    "DestAirportID": "NGO",
+    "Carrier": "ES-Air",
+    "Cancelled": false,
+    "FlightTimeMin": 454.6742272195069,
+    "Origin": "Adelaide International Airport",
+    "OriginLocation": {
+      "lat": "-34.945",
+      "lon": "138.531006"
+    },
+    "DestRegion": "SE-BD",
+    "OriginAirportID": "ADL",
+    "OriginRegion": "SE-BD",
+    "DestCityName": "Tokoname",
+    "FlightTimeHour": 7.577903786991782,
+    "FlightDelayMin": 0
+  }
+}
+```
+
+{classification-cap} is a supervised machine learning analysis and therefore 
+needs to train on data that contains the ground truth, known as the 
+`dependent_variable`. In this example, the ground truth is available in each 
+document as the actual value of `FlightDelay`.
+
+If your source data consists of some documents that contain a 
+`dependent_variable` and some that do not, the model is trained on the 
+`training_percent` of the documents that contain ground truth. However, 
+predictions are made against all of the data. The current implementation of 
+{classanalysis} supports a single batch analysis for both training and 
+predictions.
+
+
+. Create a {dfanalytics-job}.
++
+--
+Use the {ref}/put-dfanalytics.html[create {dfanalytics-jobs}] API as you can see 
+in the following example:
+
+[source,console]
+--------------------------------------------------
+PUT _ml/data_frame/analytics/model-delayed-flights
+{
+  "source": {
+    "index": [
+      "kibana_sample_data_flights"  <1>
+    ],
+    "query": {  <2>
+      "range": {
+        "DistanceKilometers": { 
+          "gt": 0
+        }
+      }
+    }
+  },
+  "dest": {
+    "index": "df-flight-delayed",  <3>
+    "results_field": "ml" 
+  },
+  "analysis": {
+    "classification": {
+      "dependent_variable": "FlightDelay",  <4>
+      "training_percent": 10  <5>
```

Comment:
It looks like perhaps the examples in this PR were taken from an older version of the Classification gist (https://gist.github.com/Winterflower/e2f226fd615a556bc1ddf9745d68df15) - in the updated version, I've bumped the training_percent to 90 to match the regression example. 
<img width="921" alt="Screen Shot 2019-11-22 at 3 12 41 PM" src="https://user-images.githubusercontent.com/4982858/69436750-48e21480-0d42-11ea-80ca-686db5fdf749.png">

Reply:
@Winterflower I used a smaller number because I wanted to capture the fact you mentioned on the ML weekly about even a smaller training percent leads to fairly great results. I figured it is clearer if the example represents this somehow. Would it be better to change back to 90%?